README.md

```diff
@@ -101,3 +104,10 @@ scrape_configs:
     static_configs:
       - targets: ['host:9999']
 ```
+
+## Building / releasing
```

Comment:
Drop these, they aren't relevant to this PR, I've already tweaked in the README a bit, I'll add these, or equivalent in there directly.